src/awx_plugins/credentials/_managed_types/terraform.py

```diff
@@ -0,0 +1,45 @@
+"""HCP Terraform managed credential type."""
+
+from awx_plugins.interfaces._temporary_private_api import (  # noqa: WPS436
+    ManagedCredentialType,
+)
+from awx_plugins.interfaces._temporary_private_django_api import (  # noqa: WPS436
+    gettext_noop,
+)
+
+from ..injectors import hcp_terraform as hcp_terraform_injector  # noqa: TID252
```

Comment:
@shvenkat-rh is this coming from our Ruff config? If yes, I'd rather disable the rule globally.

Reply:
I suppose it's this one https://docs.astral.sh/ruff/rules/relative-imports/ ?
I've disabled it on `devel`. You can now drop the suppression and rebase.
```suggestion
from ..injectors import hcp_terraform as hcp_terraform_injector
```